db/migrate/20211211182558_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Here as well. As per the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the column name should be `author` instead of `user` in the `comments` table. Please fix it.

```ruby
create_table :comments do |t|
    ...
    t.references :author, foreign_key: { to_table: 'users' }
  end
```